test/servico/LoginServiceTest.java

```diff
@@ -63,9 +64,9 @@ public void tearDown() {
     public void testeValidaUsuarioeSenha(){
 
         
-        Assert.assertTrue(LgnService.VerificarUsuarioESenha("Calebe", "123456"));
-        Assert.assertFalse(LgnService.VerificarUsuarioESenha("Calebe", ""));
-        Assert.assertFalse(LgnService.VerificarUsuarioESenha("noEcsiste", ""));
+        Assert.assertTrue(LgnService.verificarUsuarioESenha("Calebe", "123456"));
+        Assert.assertFalse(LgnService.verificarUsuarioESenha("Calebe", ""));
+        Assert.assertFalse(LgnService.verificarUsuarioESenha("noEcsiste", ""));

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Assert.assertFalse' due to existing static import 'org.junit.Assert.*'](https://www.codacy.com/app/calebepb/substituicao/file/2769922874/issues/source?bid=3308597&fileBranchId=3353664#l69)